ManatiCR/vitaminau
# VIT-24 expert services form

# Task [VIT-24](https://manati.atlassian.net/secure/RapidBoard.jspa?rapidView=104&projectKey=VIT&modal=detail&selectedIssue=VIT-24)
Service expert Webform

## Installation 

**RUN:**
- [x] ahoy drush cim -y && ahoy drush cr
- [x] ahoy drush dcdi

**- Check [descriptions](https://docs.google.com/document/d/1CiEBAoykueF8OCrT73JuCFpcu-IisLotaNFqWf541ds/edit)**

## Test
- Go to [Service expert form](http://vitaminau.docker/formularios/servicios-expertos).
- Verify beneficiaries section works correctly. 

## CA
- The webform has the requirement fields, if someone doesn't field it the form will be not send.
- You will see in the list of forms the training form.
- You can save drafts. 
- The draft preload correctly.
- The form is visible only authenticated users.
- If you don't select a code you cant send the form.
- The info automatically filled if you select a code.
- The last options is "I don't found mi code in the list"
- If you select  "I don't found mi code in the list" the unit change to a select with all the options.
